### ISSUE TITLE
Create screen  Pet Owner Profile for TabView

### ISSUE THREAD

**Issue by NadzeyaShpakouskaya** [ios-course/ironfoudation-team-project#15]
## Depends on task:

- #11 
- #20 
- #44 

## DoD:

Create a detailed PetOwner profile View

[App style guidelines](https://github.com/ios-course/ironfoudation-team-project/wiki/App-style-guidelines)

[Miro scheme with app flow](https://miro.com/app/board/uXjVMSQsET8=/?share_link_id=41046125128)

## Description

The goal of this task is to create a personal section for the pet owner's profile. This section will include a model layer for the owner and test data for the UI. The data will be saved in UserDefaults. The personal section will display personal info as a card and a button for editing personal information. On tap, the button will display a screen for editing personal information.

## Requirements

- A personal section for the pet owner's profile should be created.
- The data should be saved in UserDefaults.
- The personal section should display personal info as a card.
- The personal section should include a button for editing personal information.

**Additional details:**
The pet owner profile is a part of the main tab screen.

#### UI Example

![iScreen Shoter - 20230504104251844](https://user-images.githubusercontent.com/103530508/236154383-cc2e620f-2114-4fd1-8b97-5f78dbd995ab.jpg)

**Comment by NadzeyaShpakouskaya** [comment]
Discuss how to present the card view with personal info, when the owner hasn't filled it yet.

**Comment by NadzeyaShpakouskaya** [comment]
Should we save data  in user defaults or data will be removed when the app closed?

**Comment by iBYarchitect** [comment]
@NadzeyaShpakouskaya @julyakmita Should I create a section for pets on this screen, or did we only discuss the view and edit screens for pet owner cards?

**Comment by NadzeyaShpakouskaya** [comment]
@iBYarchitect I think we agreed only on PetOwner part for 1st sprint.
Create a separate issue for the task to add a pet section and place it in Common backlog.

**Comment by NadzeyaShpakouskaya** [comment]
As we discussed early, we can use text fields to change and update the information on CardView.
